bin/npg_pipeline_notify_delivery

```diff
@@ -132,10 +132,13 @@ if (not $mq->is_connected()) {
 }
 $mq->channel_open($channel);
 
+# 'mandatory' ensures the message reached at least one queue
```

Comment:
This setting, when enabled, also makes the message returned to the sender in case it cannot be placed. I am not sure how we can capture this on the client side. We should certainly explote the return value of the recv function. For example, when timeout is reached without getting a reply, undefined is returned.

Reply:
I cannot find info on this. Do we know behavior in this code changes and for what scenarios? I hoped this tweak only impacted message behaviour on RabbitMQ server (n.b. I think we only have to improve the situation from what we have - we can accept some imperfection at the moment...).

Reply:
The `mandatory` setting is not a change - it was tested and present in the initial deployment. I just added this comment as a housekeeping improvement while I was making other changes nearby.